requirements.txt

```diff
@@ -6,5 +6,4 @@ mpmath
 numpy
 optuna
 scipy
-pandas
```

Comment:
In what way is removing `pandas` related to improving the the docs? I suggest to leave pandas as a dependency in this PR and, if obsolete, remove it in a additional PR and then also in `mala_cpu_base_environment.yml` and `mala_cpu_environment.yml` with an explanation why `pandas` is not needed any more.

Reply:
I rewrote the installation instructions, and removed `pandas` there. Afterwards I realized I should keep it consistent by also removing it from the `requirements.txt`. It is definitely obsolete. I can revert the `requirements.txt` and make a new PR.